springboot/src/main/resources/application.yaml

```diff
@@ -0,0 +1,25 @@
+server:
+  port: 3030
+spring:
+  application:
+    name: temporal-samples
+  temporal:
+    connection:
+      target: 127.0.0.1:7233
+      target.namespace: default
+#    workers:
```

Comment:
Any reason to leave these in?

Reply:
would it be ok to leave it and put a comment there "here are some other options you could set"? 
we might not have an example for some of these options and users often ask us how to set them

Reply:
Yeah totally fine with having some example options not set by default. Just a comment explaining they are intentionally commented out it fine. Maybe we could also include TLS configuration through spring boot options?

Reply:
Great idea